modules/veteran/app/workers/veteran/vso_reloader.rb

```diff
@@ -6,8 +6,15 @@ module Veteran
   class VSOReloader < BaseReloader
     def perform
       array_of_organizations = reload_representatives
+
       # This Where Not statement is for removing anyone no longer on the lists pulled down from OGC
       Veteran::Service::Representative.where.not(representative_id: array_of_organizations).destroy_all
```

Comment:
@SlakrHakr I think this covers the part you were asking about where whether it wipes the slate clean or just removes outdated ones (seems like the latter)

Reply:
perfect! i was a bit concerned if it just wiped everything and started clean since this could lead to a situation where the data is empty for a short period of time (or for a longer period of time if the data is wiped and then there was a failure in repopulating it). this is great!